.github/workflows/c-cpp.yml

```
@@ -0,0 +1,23 @@
+name: C/C++ CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

```suggestion
    branches: [ "master" ]
    #Enable manual run
    workflow_dispatch:
```
Would be great if we could trigger this manually